message/validation/partial_validation.go

```diff
@@ -112,6 +118,8 @@ func (mv *messageValidator) validatePartialSignatureMessageSemantics(
 			return err
 		}
 
+		// MSGVALIDATIONREVIEW: since we don't expect operators to be synced on the committees' validators set,
+		// should we still keep this rule?
 		// Rule: Validator index must match with validatorPK or one of CommitteeID's validators
 		if !slices.Contains(validatorIndices, message.ValidatorIndex) {
 			e := ErrValidatorIndexMismatch
```

Comment:
@lior-blox @moshe-blox 
In `PartialSignatureMessage`, we have a `ValidatorIndex` field and there is the following rule for it:
> Validator index must match with validatorPK or one of CommitteeID's validators

But this assumes that the nodes are synced on the committees $\to$ []validators state. However, we have had discussions in which we said that we actually don't support that. Should I drop this rule?